MapleServer2/Triggers/FieldObjectTriggers.cs

```diff
@@ -124,8 +131,19 @@ public void SetRope(int ropeId, bool isVisible, bool animationEffect, byte anima
             Field.BroadcastPacket(TriggerPacket.UpdateTrigger(Field.State.TriggerLadders[ropeId]));
         }
 
-        public void SetSkill(int[] arg1, bool arg2)
+        public void SetSkill(int[] triggerIds, bool arg)

```

Comment:
could be good to keep the it named `arg2` until we figure out what it really is. Just so we can refer to the scripts referring to an arg2.